client/components/domains/map-domain-step/index.jsx

```diff
@@ -59,8 +60,7 @@ const MapDomainStep = React.createClass( {
 
 	render: function() {
 		const suggestion = { cost: this.props.products.domain_map.cost_display, product_slug: this.props.products.domain_map.product_slug },
```

Comment:
Yeah, tested it thoroughly (including a new user, empty session, throttled) and it seems OK. Probably because we load products at the start of Calypso :man_shrugging: